util/util.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
```

Comment:
Only real additions are here and below. Everything else is just using these funcs